dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -72,6 +72,8 @@
 
 LOW_MEMORY_CONDITION = 8 * 1024 * 1024 * 1024
 
+DISABLE_WARNINGS_LABEL = "disable warnings"
```

Comment:
I believed we wanted to "enable warnings" by label so that they are hiddent by default ?

Reply:
even better --- i thought you wanted them there by default

Reply:
Not really. I think it was more of @uranusjr's quest to fix them and I think he turned them on in the first place, for that reason, but I think indeed this should be more ore less dedicated effort to remove them and showing them by default when we have many of them do not play well with GitHub UI.

Reply:
yeah, i agree.  i also appreciate the desire to squash those warnings. there's a lot of noise... and if it shows up in the tests, probably a lot of it shows up for users.